zstackctl/zstackctl/ctl.py

```
@@ -8216,48 +8217,213 @@ def _dict_representer(dumper, data):
+            targets.append((management_network_ipv6.IPV4_VERSION, args.ip4))
+        if args.ip6:
+            targets.append((management_network_ipv6.IPV6_VERSION, args.ip6))
+
+        changes = []
+        for version, new_ip in targets:
+            old_ip = model[version]['ip']
+            family_name = 'IPv4' if version == management_network_ipv6.IPV4_VERSION else 'IPv6'
+            if get_ip_version(new_ip) != version:
+                error('new %s management address[%s] is not a valid %s address' % (
+                    family_name, new_ip, family_name))
+            if new_ip == '0.0.0.0':
```

Ruff (0.15.15)

[error] Possible binding to all interfaces

(S104)